components/local/confusion_matrix/src/confusion_matrix.py

```diff
@@ -40,7 +41,8 @@ def main(argv=None):
                            'If not set, the input must include a "target" column.')
   args = parser.parse_args()
 
-  on_cloud = args.output.startswith('gs://')
+  storage_service_scheme = urlparse.urlparse(args.output).scheme
```

Comment:
a few cases are 

- minio://dir/file_path 
- gs://dir/file_path  
- s3://dir/file_path
- local path "/dir/file_path"

scheme for local path is empty, all rest we consider they are `on_cloud`